src/__tests__/shared/components/input/TextFieldWithLabelFormGroup.test.tsx

```diff
@@ -70,6 +70,7 @@ describe('text field with label form group', () => {
 
       render(
         <TextFieldWithLabelFormGroup
+          id="test"
```

Comment:
Did you by chance mean this
```suggestion
          data-testid="test"
```
It seems that component is blocking the id property anyway.